### ISSUE TITLE
eeschema: moving a block and rotating it jumps back to original place

### ISSUE THREAD

**Issue by hzeller** [hzeller/kicad-source-mirror#17]
while components when moved are moving and you can rotate them as you go, a block will be tied down back to the original placed when rotated.

**Comment by thatch** [comment]
Yes, I've hit this as well.  It behaves the same in both eeschema and pcbnew -- individual work fine, blocks don't.

**Comment by thatch** [comment]
It's even worse if you copy a block then use the Paste toolbar button (which essentially puts you in the same UI as moving a block).  Then Rotate acts as cancel.
